app_test_suite/steps/scenarios/simple.py

```diff
@@ -56,7 +56,7 @@ class SimpleTestScenario(BuildStep, ABC):
 
     _APPTESTCTL_BIN = "apptestctl"
     _APPTESTCTL_BOOTSTRAP_TIMEOUT_SEC = 180
-    _MIN_APPTESTCTL_VERSION = "0.7.0"
+    _MIN_APPTESTCTL_VERSION = "0.12.0"
```

Comment:
`0.7.0` is pretty old. Is it OK if we increase this to the latest as well?

If so I think we can include it in the #news-dev post for the new release.